### ISSUE TITLE
Accessibility Violations:   The tabbable element's role 'search' is not widget roles

### ISSUE THREAD

**Issue by PallaviPattikuntapalli** [IBM/carbon-components-angular#2184]
* Location
 < div
    class="bx--search bx--search--lg bx--search--light"
    tabindex="0"
    role="search" >
   
> What browser are you working in?
   Mozilla
   
> What version of the Carbon Design System are you using?
"carbon-components": "^10.56.0",
"carbon-components-angular": "^4.59.8"

* Screenshots or code
![image](https://user-images.githubusercontent.com/108853103/177787611-315d9522-44df-477e-b7e1-630b9149c7bc.png)

**Comment by makandre** [comment]
Hi, is there an update on this issue? We just found this issue after adding the search bar component to our application.

**Comment by sant-santhiya** [comment]
Hi, is there an update on this issue?
